apps/src/studio/studio.js

```diff
@@ -1711,11 +1712,15 @@ Studio.initSprites = function () {
         if (0 === Studio.spriteCount) {
           Studio.spriteStart_ = [];
         }
-        Studio.spriteStart_[Studio.spriteCount] = Object.assign({},
-            Studio.map[row][col].serialize(), {
+        var cell = Studio.map[row][col].serialize();
+        Studio.spriteStart_[Studio.spriteCount] = Object.assign({}, cell, {
               x: col * Studio.SQUARE_SIZE,
               y: row * Studio.SQUARE_SIZE
             });
+        if (cell.sprite !== undefined) {
+          presentAvatars[Studio.spriteCount] =
+            skin.avatarList[cell.sprite + level.firstSpriteIndex];
```

Comment:
They did respect the offset, but I guess that was a bug (particularly since they weren't shown to follow it in the level editor).

Fixed that, and checked that there are no published levels that use both an offset and assigned sprites, so I'm not messing up the instructions in any existing levels by changing the displayed sprite.